.gitignore

```diff
@@ -15,3 +15,4 @@ dist/kube-policy-controller
 .go-pkg-cache
 st-kubeconfig.yaml
 *.coverprofile
+build-requirements-frozen.txt
```

Comment:
I think we can just `git rm` this file since it's a holdover from the Python days, if it still actually exists.

Otherwise let's just delete the local file :)

Reply:
Ok I hadn't put much thought into why it was there and assumed it was generated when I built things but I guess had it still on disk from previous work.